internal/events/webhooks/webhooks.go

```diff
@@ -224,7 +229,17 @@ func (wh *WebHooks) attemptRequest(sub *core.Subscription, event *core.EventDeli
 		}
 	}
 
-	req, err = wh.buildRequest(sub.Options.TransportOptions(), firstData)
+	// Create a new ffresty client from the config
+	// 1) We do not want to modify that global instance
+	// 2) We want to keep the global configuration for webhooks
+	copyFFrestyConfig := *wh.ffrestyConfig
```

Comment:
Thanks for working this all through @EnriqueL8 🙇 
Really pleased with where this landed.